src/Middleware/LocaleCookieRedirect.php

```diff
@@ -39,9 +39,9 @@ public function handle(Request $request, Closure $next)
 
         $locale  = $request->cookie('locale', null);
 
-        if ($locale !== null && ! $this->isDefaultLocaleHidden($locale)) {
+        if ($this->localization->isLocaleSupported($locale) && ! $this->isDefaultLocaleHidden($locale)) {
```

Comment:
Just revert the changes and add a check to verify the locale is not empty instead

Reply:
But then it will still thrown an exception if a user has a locale in their cookie which is no longer supported?

Reply:
Can you write a test to reproduce the issue ?

Reply:
Please triple check and have mercy, I'm not an expert at writing tests 🙈